tests/dapr_tests.mk

```diff
@@ -159,7 +159,7 @@ test-deps:
 	# In golang >=1.16 there is a new way to do this with `go install gotest.tools/gotestsum@latest`
 	# But this doesn't work with <=1.15, so we do it the old way for now 
 	# (see: https://golang.org/ref/mod#go-install)
-	GO111MODULE=off go get gotest.tools/gotestsum
+	command -v gotestsum || GO111MODULE=off go get gotest.tools/gotestsum
```

Comment:
Is `command` recognized on Windows?

Reply:
I think if you run `make`, it is in `sh` or `bash`, it would have `command`.

Reply:
I am running e2e tests and we can see if this will break on Windows.

Reply:
Pass :）

Reply:
Maybe no need to update branch, there is no conflict or any potential conflict.